api/lib/opentelemetry/trace/link.rb

```diff
@@ -15,10 +15,10 @@ class Link
 
       private_constant :EMPTY_ATTRIBUTES
 
-      # Returns the {SpanContext} for this link
+      # Returns the {SpanReference} for this link
       #
-      # @return [SpanContext]
-      attr_reader :context
```

Comment:
I feel like this should be `span_context`, although we avoid that name in `Span` due to stuttering (but perhaps we shouldn't?).